src/test/java/Homework19.java

```diff
@@ -0,0 +1,25 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest  {
+
+    @Test
+    public void Login() {
+
+        login("anna.stonaieva@testpro.io", "te$tStudent");
+    }
+        public void openPlaylist() throws InterruptedException {
+            WebElement Playlist = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+            Playlist.click();
+            Thread.sleep(1000);
+        }
+
+        public void deletePlaylist() throws InterruptedException {
+        WebElement deletePlaylistButton = driver.findElement(By.cssSelector(".btn-delete-playlist"));
+        deletePlaylistButton.click();
+        Thread.sleep(1000);
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![Screenshot_1](https://user-images.githubusercontent.com/104761482/229602285-00c13531-723e-4b18-a398-b910fdba3e0c.jpg)